jkube-kit/profile/src/test/java/org/eclipse/jkube/kit/profile/ProfileUtilTest.java

```diff
@@ -24,131 +24,123 @@
 import java.util.Collections;
 import java.util.List;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.assertThrows;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 /**
  * @author roland
  */
-public class ProfileUtilTest {
+class ProfileUtilTest {
 
     @Test
-    public void simple() throws IOException {
+    void simple() throws IOException {
         InputStream is = getClass().getResourceAsStream("/jkube/config/profiles-lookup-dir/profiles.yaml");
-        assertNotNull(is);
+        assertThat(is).isNotNull();
         List<Profile> profiles = ProfileUtil.fromYaml(is);
-        assertNotNull(profiles);
-        assertEquals(4, profiles.size());
+        assertThat(profiles).isNotNull().hasSize(4);
         Profile profile = profiles.get(0);
-        assertEquals("simple", profile.getName());
+        assertThat(profile.getName()).isEqualTo("simple");
         ProcessorConfig config = profile.getEnricherConfig();
-        assertTrue(config.use("base"));
-        assertFalse(config.use("blub"));
+        assertThat(config.use("base")).isTrue();
+        assertThat(config.use("blub")).isFalse();
```

Comment:
These  assertions (and the following) could be combined with a single assertThat DSL call ---> `assertThat(profile.getEnricherConfig())` ---> `.returns()`

Reply:
Can you tell me exactly how should I do this?
I tried to combine these assertions like

```java
assertThat(profiles).isNotNull().hasSize(4)
                .first()
                .satisfies(
                        profile -> {
                            assertThat(profile).hasFieldOrPropertyWithValue("name", "simple");
                            assertThat(profile.getEnricherConfig().use("base")).isTrue();
                            assertThat(profile.getEnricherConfig().use("blub")).isFalse();
                            assertThat(profile.getGeneratorConfig().use("java.app")).isFalse();
                            assertThat(profile.getGeneratorConfig().use("spring.swarm")).isTrue();
                        });
```

Reply:
I was thinking and more concerned about the second part (get...use), but yes something like this:
```java
        assertThat(profiles).isNotNull().hasSize(4)
            .first().hasFieldOrPropertyWithValue("name", "simple")
            .satisfies(profile -> assertThat(profile.getEnricherConfig())
                .returns(true, c -> c.use("base"))
                .returns(false, c -> c.use("blub")))
            .satisfies(profile -> assertThat(profile.getGeneratorConfig())
                .returns(false, c -> c.use("java.app"))
                .returns(true, c -> c.use("spring.swarm")));
```

Reply:
Thank you